website/i18n.js

```diff
@@ -7,7 +7,7 @@
 
 // The top-30 locales on Crowdin are enabled
 // but we enable only a subset of those
-const locales = ['en', 'ja', 'es-ES', 'pt-BR', 'ro', 'ru', 'uk', 'zh-Hans'];
+const locales = ['en', 'ja', 'es-ES', 'fr', 'pt-BR', 'ro', 'ru', 'uk', 'zh-Hans'];
```

Comment:
I don't understand the error for Typescript. Why is only 'fr' affected, should it be the case for the others too?

Reply:
it's not typescript, it's eslint since the line is too long

Reply:
just run prettier on the file and it should be happy 👍

Reply:
`yarn prettier website/i18n.js --write` should do the trick, not sure how you got double quotes 🙈